client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -200,13 +212,27 @@ class StatsPostDetail extends Component {
 				{ siteId && <QueryPostStats siteId={ siteId } postId={ postId } /> }
 
 				<div className={ postDetailPageClasses }>
-					<NavigationHeader navigationItems={ this.getNavigationItemsWithTitle( this.getTitle() ) }>
-						{ showViewLink && (
-							<Button onClick={ this.openPreview }>
-								<span>{ actionLabel }</span>
-							</Button>
-						) }
-					</NavigationHeader>
+					{ config.isEnabled( 'stats/navigation-improvement' ) ? (
+						<PageHeader
+							backLinkProps={ backLinkProps }
+							titleProps={ titleProps }
+							rightSection={
+								showViewLink && (
+									<Button onClick={ this.openPreview } primary>
```

Comment:
Any idea about whether we should use Button from `@automattic/components` or `@wordpress/components`?

Reply:
I think we should use Core than A8C components if possible.

Reply:
Update in [a3c87d7](https://github.com/Automattic/wp-calypso/pull/103039/commits/a3c87d79cd49ca73cfd6098d43207628f1c5ed94). 🙂